xml/System.Reflection/FieldInfo.xml

```diff
@@ -2096,6 +2096,8 @@
 ## Remarks  
  This method will assign `value` to the field reflected by this instance on object `obj`. If the field is static, `obj` will be ignored. For non-static fields, `obj` should be an instance of a class that inherits or declares the field. The new value is passed as an `Object`. For example, if the field's type is Boolean, an instance of `Object` with the appropriate Boolean value is passed. Before setting the value, `SetValue` checks to see if the user has access permission. This final method is a convenience method for calling the following `SetValue` method.  
   
+This method cannot be used to set values of static init-only (`readonly` in C#) fields reliably. In .NET Core 3.0 and later versions, an exception is thrown if you attempt to set a value on a static init-only field.
```

Comment:
```suggestion
This method cannot be used to set values of static, `init-only` (`readonly` in C#) fields reliably. In .NET Core 3.0 and later versions, an exception is thrown if you attempt to set a value on a static, `init-only` field.
```

Reply:
static and init-only are similar kind of qualifiers. I would either put back-ticks around both or neither.

I do not have a strong opinion about it. I would be happy to apply the suggestion if you prefer to have only init-only in back-ticks. Let me know.

Reply:
In the ECMA spec, they refer to the **initonly** attribute, but when not referring to a keyword, they use init-only as the general term. So I agree, I don't think we need the backticks here. @BillWagner?

https://www.ecma-international.org/publications/files/ECMA-ST/ECMA-335.pdf

Reply:
But we do need a comma between static and init-only.

Reply:
> But we do need a comma between static and init-only.

Fixed